pkgs/development/libraries/libgssglue/default.nix

```
@@ -1,14 +1,26 @@
-{ lib, stdenv, fetchurl, libkrb5 }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  libkrb5,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libgssglue";
-  version = "0.4";
+  version = "0.9";
 
-  src = fetchurl {
-    url = "http://www.citi.umich.edu/projects/nfsv4/linux/libgssglue/${pname}-${version}.tar.gz";
-    sha256 = "0fh475kxzlabwz30wz3bf7i8kfqiqzhfahayx3jj79rba1sily9z";
+  src = fetchFromGitLab {
+    owner = "gsasl";
+    repo = "libgssglue";
+    rev = "tags/debian/${finalAttrs.version}";
```

Debian tags don't exist in upstream repo

```suggestion
    rev = "tags/libgssglue-${finalAttrs.version}";
```

---

oh hash caching got me again